metpy/calc/basic.py

```diff
@@ -257,6 +257,77 @@ def pressure_to_height_std(pressure):
     return (t0 / gamma) * (1 - (pressure / p0).to('dimensionless')**(Rd * gamma / g))
 
 
+@exporter.export
+@check_units('[length]')
+def height_to_geopotential(height):
+    r"""Computes geopotential for a given height in meters
```

Comment:
One of the typical styling things is to have this first line as an imperative and end in a period. Also, single quotes everywhere is what we've standardized on. Also, since we have the unit framework, we try to leave units out of the docstrings unless absolutely necessary. So can you change to:
```python
'''Compute geopotential for a given height.
```

Reply:
In basic.py all the docstrings contained double quotes so I replaced them in all the docstrings. Also changed to imperative and a period in the end.